roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -107,30 +107,27 @@ def get_var(self, *keys, **kwargs):
                 return kwargs["default"]
             raise OpenShiftCheckException("'{}' is undefined".format(".".join(map(str, keys))))
 
-        convert = kwargs.get("type")
+        convert = kwargs.get("convert", lambda x: x)
```

Comment:
Oh my, even JSON... I forgot about that ;)

The best would be to exterminate the INI format and command line flags in the next Ansible version, and made convert it all to Haskell -- hhaha, or maybe not :P